cmd/tailscale/cli/web.go

```diff
@@ -83,6 +85,35 @@ var webArgs struct {
 	cgi    bool
 }
 
+func tlsCertFromEnvironment(listen string) (*http.Server, error) {
```

Comment:
it's named as if it returns a "tlsCert" but it returns a Server. And it returning (nil, nil) is worth documenting.

Reply:
Renamed tlsConfigFromEnvironment and made it return tls.Config.